### PR TITLE
CircleCI config, dependency update

### PR DESCRIPTION
**Pull request by jescalan**
A couple small trailing changes that I forgot to push to the previous PR. Sorry 😞

### CODE REVIEW

**Comment by mitchellh** [comment]
We have to backport this too yes?

File: .circleci/config.yml [hunk 1]
```diff
@@ -439,7 +439,9 @@ workflows:
             branches:
               only:
                 - main
-      - algolia-index:
+  website:
+      jobs:
+        - algolia-index:
```
Comment (krantzinator) [inline .circleci/config.yml:444]:
```suggestion
    jobs:
      - algolia-index:
```

Comment (krantzinator) [reply .circleci/config.yml:444]: The spacing didn't seem to stick from your local to this PR?

Comment (jescalan) [reply .circleci/config.yml:444]: I don't know how i can keep doing this wrong i am baffled

**Review by krantzinator — APPROVED** [review]

**Comment by krantzinator** [comment]
You have defeated the YAML! For now... 😄

**Comment by mitchellh** [comment]
You only think you defeated the YAML.

**Review by briancain — APPROVED** [review]

**Pull request merged**